app/models/errors/httpErrors.scala

```diff
@@ -20,14 +20,16 @@ sealed trait HttpError {
   def message: String
 }
 
-case class BadRequestError(code: String, message: String) extends HttpError
+case class BadRequestError(code: String, error: String) extends HttpError {
+  override val message: String = s"The server you connecting to returned an error. [BadRequest]- RESPONSE status: $code, message: $message"
```

Comment:
Should the _$message_ at the end of this line be _$error_? Also for consistency's sake, if the other types of error use the variable name _errorResponse_ then maybe this _error_ variable should also be name _errorResponse_?